### PR TITLE
MidFinal CI/CD

### PR DESCRIPTION
**Pull request by nadeemjazmawe**
Thank you for your contribution.
Before submitting this PR, please make sure:
- [ ] PR description and commit message should describe the changes done in this PR
- [ ] Verify the PR is point to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [ ] Latest Code from master or specific release branch is merged to your branch
- [ ] No unwanted\commented\junk code is included
- [ ] No new warning upon build solution
- [ ] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [ ] Sanity Tests are successfully executed for Existing Functionality
- [ ] After creating pull request there should not be any conflicts
- [ ] Resolve codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments

### CODE REVIEW

File: Extensions/DotnetDependencies/aspnetcore-runtime-6.0.6-win-x64.exe
Comment (Maheshkale447) [inline Extensions/DotnetDependencies/aspnetcore-runtime-6.0.6-win-x64.exe:1]: Please replace it with .Net 7 dependencies, version 7.0.9

File: Extensions/DotnetDependencies/windowsdesktop-runtime-6.0.6-win-x64.exe
Comment (Maheshkale447) [inline Extensions/DotnetDependencies/windowsdesktop-runtime-6.0.6-win-x64.exe:1]: Please replace it with .Net 7 dependencies, version 7.0.9

File: Ginger/GingerInstallerScriptGithub.iss
Comment (Maheshkale447) [inline Ginger/GingerInstallerScriptGithub.iss:1]: Looks script is not latest from master as its referring to .net 6, please use latest script from master

File: Ginger/GingerRuntime/Dockerfile
Comment (Maheshkale447) [inline Ginger/GingerRuntime/Dockerfile:1]: Please update dotnet version here to .net 7 as well

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/GingerRuntime/Dockerfile
Comment (nadeemjazmawe) [reply Ginger/GingerRuntime/Dockerfile:1]: I'm  aiming to make the changes for all Dockerfiles at once, For this docker file I have done the change.

**Pull request merged**